pre-push.sh

```diff
@@ -0,0 +1,13 @@
+git stash -q --keep-index
+
+
+# Tests
+npm test
+
+RESULT=$?
+
+git stash pop -q
```

Comment:
You need to check if anything was actually stashed before popping. Otherwise unwanted code could be popped instead